hasher-matcher-actioner/terraform/api/main.tf

```diff
@@ -104,77 +104,6 @@ resource "aws_iam_role_policy_attachment" "api_root" {
   policy_arn = aws_iam_policy.api_root.arn
 }
 
-
-# Authorizer API Lambda
```

Comment:
nice! can you delete the python code too?

Reply:
absolutely!